### PR TITLE
added contribute index

### PR DESCRIPTION
**Pull request by frunika**

### CODE REVIEW

File: docs/contribute/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Getting Started
+
+> Become a part of OpenTrafficCam
+
+We are happy if you contribute your own code to OpenTrafficCam.
+This can be bug fixes in the current code, or even completely new stuff.
+It is best to start at [GitHub](github.md) to get a first overview.
+
```
Comment (martinbaerwolff) [inline docs/contribute/index.md:8]: Add a section that even testing the framework and creating issues in the Github repositories or updating the [documentation](https://opentrafficcam.org/contribute/documentation/) is appreciated?

**Review by martinbaerwolff — CHANGES_REQUESTED** [review]

File: docs/contribute/index.md (hunk 1 above)
Comment (frunika) [reply docs/contribute/index.md:8]: This way?

File: docs/contribute/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Getting Started
+
+> Become a part of OpenTrafficCam
```
Comment (martinbaerwolff) [inline docs/contribute/index.md:3]: > Become a part of  Open**Traffic**Cam
?

File: docs/contribute/index.md (hunk 1 above)
Comment (martinbaerwolff) [reply docs/contribute/index.md:8]: Maybe also issues with code, not only with documentation?

**Review by martinbaerwolff — CHANGES_REQUESTED** [review]

File: docs/contribute/index.md (hunk 2 above)
Comment (frunika) [reply docs/contribute/index.md:3]: no...

File: docs/contribute/index.md (hunk 1 above)
Comment (frunika) [reply docs/contribute/index.md:8]: It's all on the GitHub page. I think we don't need it twice.

Comment (martinbaerwolff) [reply docs/contribute/index.md:8]: But whats with the OTC users who just download the framework without ever visiting the github page?

Comment (frunika) [reply docs/contribute/index.md:8]: They are not reporting any code issues ;)

**Review by martinbaerwolff — APPROVED** [review]

**Pull request merged**